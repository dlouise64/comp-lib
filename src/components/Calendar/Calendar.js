import React from 'react';
// import PropTypes from 'prop-types';

import './Calendar.css';

class Calendar extends React.Component {
  render() {
    return (
      <div className="calendar">Calendar</div>
    );
  }
}

Calendar.propTypes = {
};

Calendar.defaultProps = {
}

export default Calendar;
