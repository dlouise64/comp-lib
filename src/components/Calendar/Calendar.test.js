import React from 'react';
import renderer from 'react-test-renderer';
import Calendar from './Calendar';

test('<Calendar /> renders correctly', () => {
  const tree = renderer.create(<Calendar />).toJSON();
  expect(tree).toMatchSnapshot();
});
