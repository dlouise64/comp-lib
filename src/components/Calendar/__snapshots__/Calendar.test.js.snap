// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Calendar /> renders correctly 1`] = `
<div
  className="calendar"
>
  Calendar
</div>
`;
